browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -251,6 +267,15 @@ class SlideShowPresenter {
 		return canvas;
 	}
 
+	private exitSlideshowWithWarning() {
+		const transitionParameters = new TransitionParameters();
+		transitionParameters.context = this._slideRenderer._context;
+
+		new SlideShow.StaticTextRenderer(transitionParameters).display(
+			'Click to exit presentation...',
```

Comment:
Please always use translated strings:
`_('Click to exit presentation...')`

Reply:
Done.